backend/app/Http/Controllers/LessonController.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Lesson;
+use Illuminate\Http\Request;
+
+class LessonController extends Controller
+{
+    public function index()
+    {
+        return Lesson::all();
+    }
+
+    public function create(Request $request)
+    {
+        $fields = $request->validate([
+            'category_id' => ['required'],
+            'character' => ['required', 'unique:lessons'],
+        ]);
+
+        $user = Lesson::create([
+            'category_id' => $fields['category_id'],
+            'character' => $fields['character'],
+        ]);
+
+        $response = [
+            'user' => $user,
+            'status' => 'Lesson Created',
+        ];
```

Comment:
Improve naming conventions that are related to corresponding Model.
```suggestion
        $lesson= Lesson::create([
            'category_id' => $fields['category_id'],
            'character' => $fields['character'],
        ]);

        $response = [
            'lesson' => $lesson,
            'message' => 'Lesson Created',
        ];
```